.github/workflows/build-and-deploy-prod.yml

```diff
@@ -44,6 +44,9 @@ jobs:
                 VITE_TRANSLATIONS_CDN_URL: ${{ vars.VITE_TRANSLATIONS_CDN_URL }}
                 VITE_TRACKJS_TOKEN: ${{ vars.VITE_TRACKJS_TOKEN }}
                 VITE_APP_VERSION: ${{ github.ref_name }}
+                VITE_GROWTHBOOK_DECRYPTION_KEY: ${{ vars.VITE_GROWTHBOOK_DECRYPTION_KEY }}
+                VITE_GROWTHBOOK_CLIENT_KEY: ${{ vars.VITE_GROWTHBOOK_CLIENT_KEY }}
+                VITE_RUDDERSTACK_KEY: ${{ vars.VITE_RUDDERSTACK_KEY }}
```

Comment:
@ali-hosseini-deriv could you please help to add this variables to the p2p repo? 🙏

Reply:
these are provided in the github secrets @shayan-deriv